src/lib/containers/ModalDownload.tsx

```diff
@@ -164,26 +164,16 @@ export default class ModalDownload extends React.Component<
                 </div>
               </div>
             )}
-            <hr />
             <div style={{ textAlign: 'right' }}>
-              <a
-                id="cancelBtn"
-                onClick={this.props.onClose}
-                type="button"
-              >
+              <Button variant="default" onClick={this.props.onClose} style={{ marginRight: '5px' }}>
                 Cancel
-              </a>
-              <button
-                id="submitBtn"
-                style={submitBtn}
-                className="SRC-primary-background-color SRC-roundBorder SRC-whiteText"
-                type="submit"
-              >
+              </Button>
```

Comment:
Initially moved these into a Modal.Footer section, but then realized author was using the Form submit to handle the event.